big_tests/tests/graphql_muc_light_SUITE.erl

```diff
@@ -537,6 +559,214 @@ user_blocking_list_story(Config, Alice, Bob) ->
                     <<"entity">> => RoomBin}],
                  get_ok_value(?GET_BLOCKING_LIST_PATH, Res5)).
 
+%% Domain admin test cases
```

Comment:
A lot of tests are almost identical to the ones for the global admin. Could you reuse code instead? The story function could just have one more argument - the room name, as I think this is the only thing that is different.